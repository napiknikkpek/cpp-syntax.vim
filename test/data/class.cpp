class Test;
class Test {};
struct Test : Base {};
class Test : public A, private B, protected A::B::C {};
class Test {
  class Inner : public Base {};
};
