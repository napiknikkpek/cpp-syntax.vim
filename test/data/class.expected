aaaaa.tttt.
aaaaa.tttt....
aaaaaa.tttt...tttt....
aaaaa.tttt...aaaaaa.t..aaaaaaa.t..aaaaaaaaa.t..t..t....
aaaaa.tttt..
..aaaaa.ttttt...aaaaaa.tttt....
..
